Cab share/Login/LoginView.swift

```diff
@@ -101,7 +116,31 @@ struct LoginView: View {
             }
             .navigationBarHidden(true)
             .preferredColorScheme(.dark)
+//            .onAppear{
+//                Auth.auth().addStateDidChangeListener{ auth, user in
+//                    if user != nil {
+//                        userIsLoggedIn.toggle()
+//                    }}
+//            }
+        }
+    }
```

Comment:
Delete this one if we don't need it

Reply:
i already deleted it. maybe how i told you, there are some file that are not updated